source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,23 @@
+=================
```

Comment:
Nitpick: the line length is much longer than the 80 characters that our team wants to limit our docs to when possible. Could you hard wrap the long lines when possible?

Reply:
Fixed